### ISSUE TITLE
Meeting Recap 02-24

### ISSUE THREAD

**Issue by lmcneil7** [lmcneil7/teen-titans#38]
One of these days, this might be put up on GitHub on Monday and not multiple days later at 4 am.

### Next Meeting: 02 March 2020
_Recap:_

**Update on Markup Process**
- Shannon was able to do one episode, and Amber has started on her second episode. I plan to work on it tonight since I have less work to do for my Tuesday classes.
- **Due Date: Feb 29**
    - We want to do it by then, and Fiona advises as such. If we come to realize we don't have it done, we need to be done by **Mar 6!**

**Movie Night**
- We're still meeting up this Saturday! The time is 5:15ish depending on when Amber's meeting with Dr.B ends. We plan to split this up on two days.
- _Sunday:_ What time do we want to meet up? While the group said Lauren should choose, I don't make decisions so that we will see. I'm saying maybe _2 pm_ or _3 pm_.

**Maps**
- Since Undertale didn't create their maps either, I mentioned that it's not what I wanted to do. While both Pokemon and Undertale are cool projects, I wasn't able to find a high image to use for image mapping/iframes.
    - Fiona mentioned photoshopping the image I did find? _Consider photoshop as a way to map._
- While I don't want to do San Francisco, KML and GIS are both possible options. I like GIS more as it would help with the layering we want to do.
    - I forgot to mention this in the meeting, but from research, people suggested using more unknown cities that fit the outline that I want and modify the map instead of using a known metropolis.
    - San Francisco would be a lot of interpretations.
- Ideally, I want to make a fictional map. I don't know how to do this. I looked online and found some websites where you can create maps, but I wouldn't know how to implement it on the site.
_Websites:_
 https://watabou.itch.io/medieval-fantasy-city-generator/devlog 
https://azgaar.github.io/Fantasy-Map-Generator/
http://mewo2.com/notes/terrain/
- Drawing is the hardest idea that I have, but using the images I have as a reference, I think it could be a last-resort idea. I would be creating the image to do image mapping.
- **Action on Lauren to have a concrete idea by the next meeting!!!!!!!!**

**Schema Revisions**
- I mentioned that consistent ids are essential, and Shannon has already kept that in mind as she marked up episode 3. While we might be able to input this into Schematron, right now, we check each other's markup to make sure they match if we are dealing with the same locations. 
- We won't worry about indicating when settings have two names in `<location>` elements.
- Shannon was confused about the `<action>` tag, but she said she thinks she got it. I will look over her episode 3, mostly for `<setNote>,` but I'm sure it's okay.

**editing the README.md file**
- **Action on Lauren**
- ### I forgot to ask, but if you have anything I should add, let me know. Orrrrrrrr you could make a branch pull request!

**Schematron**
- We want to implement it sooner! No specific date when we want to do this, but we will be looking into before we start in class (Mar 18).
- Could put action one person to start it, and then the other two add on with their ideas.
- will think more about implementation after we finish markup

**Defining the research question**
- Sorry for putting you on the spot @frabbitry, but let me know if you have any more questions about the research question.
- The base map is locations from the tv show. Then, we will graph the titans' movements throughout season 2. We also are thinking about linking out to scripts via a clickable map.
- I didn't mention this in the meeting, but the comics won't be on the map but will have a separate graph, also mapping titans' movement. The comic graph is for analysis purposes to keep the research question from the fall semester relevant to the project (comparison between comics and the tv series).
    - _When I say graph, I mean a line graph. Since we've only done SVG, that's what I'm imagining we will use._
- screwmeneutics spoke to me and is possibly the closest thing to what I'm imagining for the map. 
    - _Problem: I tried to find visuals or even the StageGraph tool itself, but unless this man decided to start wood making, searching his name in Google is not giving me helpful data._

**Unmasking the Teen Titans**
- If we finish the markup at the time that we want, we will start cleaning up the website.

**Other:**
Amber said she's going to run network analysis on the site, using the first season's markup. Keep us posted @amberpeddicord on how that goes, and we will look into it more with season 2 after the markup gets completed.

This is a lot of reading, so in the future, I'll try to keep them shorter! It's just a lot of the information I don't get into detail at the meeting, so I want to make sure it's clearer with the minutes. Also, I find if I write everything verbatim when I remember, I'll have it to reference when I ultimately forgot.  Overall, I think I've touched on what we really need to do, but let me know if you think I'm missing something. As usual, the meeting agenda will be up next Monday! I hope everyone has a great day!

**Comment by ebeshero** [comment]
@lmcneil7 That’s an exciting update as you’re thinking up visualization strategies. Here are some comments/ideas I had while reading it: 

1) You lot need Schematron in your lives just now to make sure of consistency in all your files, and also to be super clear in your brainwaves on things that *could easily become* inconsistent in your files. You may need a review of this and some help—since we don’t have a long unit on Schematron and it can be tricky to write, so I say, grab me or @frabbitry or @alnopa9 to work with you on specifically this! It probably won’t take long and may be illuminating at this stage.

2) Yeah, I think Steve Ramsay took down any images from StageGraph that he ever had up and I don’t know why. @frabbitry was noticing that too. I remember when he visited us (he came to speak to the Pittsburgh and Greensburg DH classes like 3 years ago?), he showed an image or two from the project in his slides—they are basically network graphs that associate characters with locations in little clusters. And you will be learning how to make those yourself in...about a week! I am excited to see what kinds of correlations you will be able to plot in network graphs. Also, here is an unusual idea, but is probably just fine, and may even be welcome and flattering to Steve Ramsay. Since he did come to visit our DH classes years ago I am sure he remembers us, and he may be interested to hear about your project. You could write to him directly by e-mail and ask him if he could show you some of the Stage Graph visuals and explain why you wanted to see them based on your project! Steve Ramsay is at U Nebraska Lincoln. I’ll go look that up and share it here in a second. 

3) I think you have some great ideas for generating images for maps and improving what is available. If the images you have are kind of small simple low-res diagrams, consider converting those to SVG (there are great programs like Inkscape to help do this) so they scale nicely! There are a few approaches to doing that, and one that occurred to me was actually generating SVG from tracing over the contours of an image. If you have it in SVG, there will be ways to work with it as an image map that are different from what Pokemon and Undertale did. I don’t remember but Bloodborne Lore’s image map might be SVG based. Anyway, I posted an issue with some guidance on SVG image mapping last year—I will find it and link it here.

**Comment by ebeshero** [comment]
Stephen Ramsay on GitHub! (very active recently: send him a message or ping from an issue?) https://github.com/sramsay 
Or e-mail (from his GitHub repo): sramsay@protonmail.com 
@lmcneil7 @frabbitry

**Comment by ebeshero** [comment]
Here's the issue I posted last Spring with much much much to say about how to do **responsive image mapping** with SVG and CSS: Lots of links here with tutorials and explanations:
https://github.com/ebeshero/DHClass-Hub/issues/652 

@lmcneil7 @frabbitry

**Comment by lmcneil7** [comment]
A response!

1. I just remembered I had said nothing on comic markup this entire time. Please, someone, remind me to mention it to the next meeting agenda!
1. Thanks, @ebeshero, for the info about @sramsay! I am disappointed about the visuals, though; while I understand a lot of what he was going for, it would make more sense for the implementation if I had any idea how he graphed any of it. Seeking a peek at his website, it wouldn't make sense to write an issue on his repo (my original plan), so I'm going to write one here and hope for the best! It's he was talking about graphing characters' movements and interpretation of data (large parts of _Teen Titans_), and I thought it would be great to understand him more. I hope he's up to help a college student understand "screwmeneutics." 
1. You are 100% right about the Schematron. While I'm not too sure about all the details yet on what we're going to use for Schematron for (a lot of it is location), I will bring it up at the next meeting. Time to get the team members brainstorming, so we can ultimately beg Fiona and Alyssa for some of their mass wisdom.
1. I swear you can never stop learning new things in the world of coding! I was not aware that you could use SVG to resize graphs.

**Comment by frabbitry** [comment]
@lmcneil7 @ebeshero 
It would also be very doable to make network graphs to correlate characters to locations... keep me updated on any correspondance you have with Ramsay; I'd be very curious to see what he has to say. 

Just a minor point about interpretation: if you do go for a _map_ map (i.e. GIS or KML), you're going to be doing interpretation. It's kind of par for the course, because at the very least you're making an assumption about the work that is Teen Titans season 2 that it could be represented in some way by putting characters on a map. You're going to have to make interpretations no matter what kind of map you use, even if it's a random one of a place that looks kinda like Jump City. I mean, really, if you think about it, research in and of itself is interpretation. 

I'm really, really working very hard at restraining myself from throwing a bunch of literary theory at you on this subject, but it's there. 

If you WERE to go with San Francisco, yes, you'd be making an interpretation. You'd be SAYING SOMETHING or PUTTING FORTH A THEORY about what the writers of Teen Titans were SAYING with their story, in that they were possibly SAYING SOMETHING about San Francisco. And this is research! Possibly even cultural criticism! @smdunn921 and @amberpeddicord could write capstones about stuff like this! 

If you have even one source that puts even one location from Jump City on the map of SF, then you have a start. You have a basis for making the interpretation. From there, yes, you'd have to be "guessing" in a way. However, if it makes you feel better, one of the advantages to GIS is that you can work with areas and not just strict points. In other words, you can create a shape that represents an area and put it on the map instead of just a single point. So maybe if you're not totally sure about where a place would be you could make an approximate area and have the characters go there? I don't know; it's just an idea.
